### ISSUE TITLE
[subviews] can't create a mage

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#995]
BG1 plain mage chargen fails when/just before trying to construct a scrollbar for the spell list:
```
Attribute Error:
[Python/ERROR]: SystemError: ../Objects/longobject.c:998: bad argument to internal function
[Python/ERROR]: Traceback (most recent call last):
[Python/ERROR]:   File "../gemrb/GUIScripts/bg1/GUICG7.py", line 50, in OnLoad
[Python/ERROR]:     LUSpellSelection.OpenSpellsWindow (Slot, TableName, Level, Level, KitValue, 1,False)
[Python/ERROR]:   File "../gemrb/GUIScripts/LUSpellSelection.py", line 131, in OpenSpellsWindow
[Python/ERROR]:     SpellsWindow.CreateScrollBar (1000, {'x' : 325, 'y' : 42, 'w' : 16, 'h' :252})
[Python/ERROR]:   File "../gemrb/GUIScripts/GUIClasses.py", line 183, in CreateScrollBar
[Python/ERROR]:     view = CreateView (control, IE_GUI_SCROLLBAR, frame, CreateScrollbarARGs(bam))
[Python/ERROR]: AttributeError: Error retrieving image from list
```

Some binding function doesn't clear errors, so PyErr_Occurred() craps out in CreateView. It's set already on entry, so it goes wrong earlier (notice the SystemError), this just happens to be the first time we check for errors afterwards.

**Comment by bradallred** [comment]
the problem is that `SpellsWindow` is invalid

**Comment by lynxlynxlynx** [comment]
Hmm, could we improve the diagnostic? There's no reason for the SpellsWindow to be invalid there, but it could be some other.

**Comment by bradallred** [comment]
there is a `Close()` call right about that line. I assume we hit that.

I'm not sure of a good way to improve the diagnostic. I believe it happens in `PyArg_ParseTuple`

**Comment by bradallred** [comment]
deleting the `Close()` call does "fix" the issue. Obviously whatever that close is there fore needs to be discovered and addressed in a more sane way.

**Comment by lynxlynxlynx** [comment]
Huh, that's some odd code, didn't even notice it. Of course it turns out it's my "fault" from 2009. 😁 This was done when merging bg1 and bg2 scripts, but so far looks fine. Will fix #994 first to be able to test properly.
